.travis.yml

```diff
@@ -1,65 +1,77 @@
-sudo: false
-env:
-  global:
-    # travis encrypt AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID
-    - secure: "c4QsffnEeuQHFCKZEILmp744KoCXrHuOfTVrUDeuTa/1raBB8LxIda6uJj1ucq4BeovQ5gmy6Xu9aw1s8nUD1ZmuwuE/N4GpqXKa1lnFe31ry8/NRuEChf/jvPi0Jv91MXMmd4moXLUh7sNXVF4Xw5VZumXC6EhvsmwhpA8UoPvs3lNl2Tr4XIET8JN37R5hElX5jZNx89RglWeg8dHuSKA1t4BJFcfcYSCpaodyrBxejMGSyV2TCz9CTgh4z4iedy+Su2tvZCmyhU0sdgqX+FMUO2KeISM0ak+F91YBAX8DApQ0Y1QeZdqN80WhLbIogoIODdmiQzrAwbjqlJz6Qp53j3UTo+pAz1dpW5g9MjA3w1McnO6vrPmy/Oupkr2Lb4/uXvIs6bW5SlrvXqze+cbvRXEoKx4GUKyKlgiDW4lae1ZtrpJoVD18TUCi6NOG2oYCz9wg2b99CLTUVy4Ay9+95QnYq+TksWNThe1ZGLP9KHTJqHkw7l8QtXYiMoysNLCfhtWItY9RnvVCRBgD4mHs58Qg/HeciH8g3VIRHtOO/LA5ID5+ZdNGrf1PxXIVsnknzBH7amm5KkPOq0lmLeLstqYhkhv5aDPdPvYT/W+5IvFVmQhVKxvoRVw+XDYtLpdgBLXT2tdueig4YWAS7IrWvMEGz70aonJHewm/IT0="
-    # travis encrypt AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY
-    - secure: "b5DPZlVJ8wVMZBhXm6WkKXRg+Gp5nWycIVzZHsGrYPStd/QYbC76qoHAo6JgocpeiPNeLxD1F/E72v3qNL0ksM2mMJuJk9uEPV4B3RdguGbhbdhffj1KaSqFZz64O0AhRkNtzyj62ezM3iYRWFkE7ysdUDgXUmv76s8Xl65piS2F1ZVLC5xPX9aSsZPciLL3AiZzMgolGnjHTrvnny1mXFiyff5qhz7Ntr51VUeDiMbJT96OplBsF7A2ZnkIaKMjwQHgCjMUJpFue+yXFdqxOQieOMQyNHqIe783P+2spxiuPgamvwl9xI6VGOA3HFuLZSQ9UGC+xLFepRr6jH8H4ipqrDO6/6JgENVORNQbREzu5wjJSlVea19Utv+dGQvK+c/xoFHpH5ZFg9lZQLCSGwRtMYGYlrc1JODigMr81S0QTYpxgoldCow71za1xPXu9699uctYyduhK4LjjJQRFOwIdcJG+gyKxpApqKd22AFQ+I/dHfEX1nAAdpVF3Ap4eFSUfH2ENKNDCFhUsBLF0czcxaXDIRwGdYG21D2NDaJyozME54wBz3yiJjOnS7b9lc16Sq/gZUqdWqGlr7AVqYhz0QKUPfSnc+ql3DCs7bNoN4V5FYheQL9+mLuEcyz7ubz8VotTlR5dcb8vtQAoLPLk1rUBAeDe1/2MkdsH75E="
+stages:
+  - test
+  - name: build
+    if: head_branch = master
+
+# Defaults
 language: node_js
-node_js:
-- '8'
-before_install:
-- npm i -g npm@6
-install:
-- npm ci
+node_js: '8'
+
+# Install latest npm & dependencies
+before_install: npm i -g npm@6
```

Comment:
Does the 6 here conflict with the 8 above?

Reply:
Nope, this is **npm** 6. Node 8 comes with npm 5 but we want 6 for `npm ci`. This hasn't changed from the previous config. 

This is all moot if we upgrade to Node 10 as in this PR https://github.com/biglotteryfund/blf-alpha/pull/1162 as Node 10 is bundled with npm 6.

Reply:
This is what I get for doing code reviews before coffee. Carry on!